### ISSUE TITLE
Writing emissions.csv to non-default output_dir, emissions are always 0.0

### ISSUE THREAD

**Issue by enjeeneer** [mlco2/codecarbon#101]
* CodeCarbon version: 1.0.0
* Python version: 3.7.9
* Operating System: MacOS

### Description

I've been trying to write emissions.csv to a chosen `output_dir` (rather than the default working directory) to track emissions data across experiments in the same place. When I do so, I always get emissions of `0.0`, but when I write to the default directory this is not the case and I get a small non-zero result. I've found this to be happen across many different experiments.

In the code below, `e_path` is just the path to the directory I am trying to write to. This may be a mistake on my end, in which case any help is much appreciated, otherwise it looks like a bug.

### What I Did

```
from codecarbon import EmissionsTracker

mnist = tf.keras.datasets.mnist

(x_train, y_train), (x_test, y_test) = mnist.load_data()
x_train, x_test = x_train / 255.0, x_test / 255.0


model = tf.keras.models.Sequential(
    [
        tf.keras.layers.Flatten(input_shape=(28, 28)),
        tf.keras.layers.Dense(128, activation="relu"),
        tf.keras.layers.Dropout(0.2),
        tf.keras.layers.Dense(10),
    ]
)

loss_fn = tf.keras.losses.SparseCategoricalCrossentropy(from_logits=True)

model.compile(optimizer="adam", loss=loss_fn, metrics=["accuracy"])

tracker = EmissionsTracker(project_name='test', output_dir=e_path)
tracker.start()
model.fit(x_train, y_train, epochs=50)
tracker.stop()
```

**Comment by LiamConnell** [comment]
Hi @scottjeen thanks for bringing this up. I just replicated the problem on my machine and I see the same thing.

**Comment by LiamConnell** [comment]
@scottjeen I think the problem is actually due to a bug with the scheduler. I've submitted a PR that is now pending review. In the meantime, could you try adding `measure_power_secs=3` like so:

`tracker = EmissionsTracker(project_name='test', output_dir=e_path, measure_power_secs=3)`

I think you will get a non-zero power/emissions reading

**Comment by scottjeen** [comment]
@LiamConnell Thanks for your reply, unfortunately this doesn't solve the problem on my machine, `0.0` is still written to emissions.csv. When I run again without passing an `output_dir` a non-zero reading is produced, so it seems that the issue is directly related to `output_dir`. Sorry to bring bad news!

**Comment by LiamConnell** [comment]
@scottjeen Thanks. I looked into the `intel_power_gadget_log.csv` problem yesterday, and I do get a file of that name written into the `output_dir` that I specify. 

Can you try adding these lines (after you import codecarbon) and pasting in the logs you get: 

```import logging
logging.basicConfig(level="INFO")```

**Comment by vict0rsch** [comment]
@LiamConnell do you believe this is fixed?

**Comment by LiamConnell** [comment]
@vict0rsch Yes. I'm closing. The problem was almost definitely do to the scheduler problem which is resolved
